README.md

```diff
@@ -1,2 +1,97 @@
-# Hello-Microverse
-Microverse - set up project
\ No newline at end of file
+# HTML & CSS3 Course
+
+If you are not familiar with linters, read [root level README](../README.md).
+
+## Set-up GitHub Actions
+
+Please do the following **steps in this order**:
+
+1. In the first commit of your feature branch create a `.github/workflows` folder and add a copy of [`.github/workflows/linters.yml`](.github/workflows/linters.yml) to that folder.
+   - **Remember** to use the file linked above
+   - **Remember** that `.github` folder starts with a dot.
+2. **Do not make any changes in config files - they represent style guidelines that you share with your team - which is a group of all Microverse students.**
+   - If you think that change is necessary - open a [Pull Request in this repository](../README.md#contributing) and let your code reviewer know about it.
+3. When you open your first pull request you should see the result of the GitHub actions checks:
+
+![gh actions checks](../assets/images/gh-actions-html-css-checks.png)
+
+Click on the `Details` link of each action to see the full output and the errors that need to be fixed:
+
+![gh actions failing checks](../assets/images/gh-actions-html-css-failing-checks.png)
+
+## Set-up linters in your local env
+
+**Note**: The `npm` package manager is going to create a `node_modules` directory to install all of your dependencies. You shouldn't commit that directory. To avoid that, you can create a [`.gitignore`](https://git-scm.com/docs/gitignore) file and add `node_modules` to it:
+
+```
+# .gitignore
+node_modules/
+```
+
+### [Lighthouse](https://developers.google.com/web/tools/lighthouse)
+
+An open-source, automated tool for improving the quality of web pages. It has audits for performance, accessibility, progressive web apps, SEO and more.
+
+You can get the Lighthouse report by any of the following ways:
+
```

Comment:
- [ ] Good job setting up the project, however please use the correct readme file, currently you have added the linters config readme, The readme file is the documentation of our projects, It contains details about our project like the project title, Key features, a description and so on, please use [this template](https://github.com/microverseinc/readme-template) but remember to customize it to your project.

Copy the Readme.md file from the above link and use it as a template, fill in the details to customise it to your projec such as the description, tech stack , title, and so on, add a MIT license file to your project's root folder and linkit to the readme, remove the microverse logo too :+1: 
here is a pic of the readme template :point_down: 
![image](https://user-images.githubusercontent.com/97521557/211616860-0fd6e5ab-383d-4a07-97b2-fd725b740cdb.png)